grakn-engine/src/main/java/ai/grakn/engine/data/RedisWrapper.java

```diff
@@ -150,7 +150,7 @@ public RedisWrapper build() {
             if (useSentinel) {
                 jedisPool = new JedisSentinelPool(masterName, uriSet, poolConfig, TIMEOUT);
             } else {
-                String uri = uriSet.stream().findFirst().get();
+                String uri = uriSet.iterator().next();
```

Comment:
Maybe `Iterables#getOnlyElement` is better here? I guess it depends if we expect there to be more than one or not.

Reply:
We can expect more than one so I can't use`Iterables`